age/front/react/README.md

```diff
@@ -0,0 +1,27 @@
+# React + TypeScript + Vite
```

Comment:
I think we should either:
- Rewrite it with our own content; or
- Delete it

Since we already have a README for this example, I think we can just delete it

Reply:
(and maybe add the instructions to run it in the age readme, or add "global" instructions in the repository readme directly)